src/main/scala/mesosphere/mesos/TaskBuilder.scala

```diff
@@ -197,6 +196,7 @@ object TaskBuilder {
     // args take precedence over command, if supplied
     app.args.foreach { argv =>
       builder.setShell(false)
+      builder.setValue(argv.head)
```

Comment:
Because of https://github.com/apache/mesos/blob/0.22.1/src/docker/docker.cpp#L451 -- we should only do this if this task uses the Mesos containerizer.